languages/ruby/.rubocop.yml

```diff
@@ -3,5 +3,6 @@ AllCops:
   Exclude:
     - '**/*~'
     - 'bin/oso'
+    - 'lib/oso-oso.rb'
```

Comment:
Instead of excluding this file from `AllCops`, let's just exclude it from `Naming/FileName`:

```yaml
Naming/FileName:
  Exclude:
    - 'lib/oso-oso.rb'
```